presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -298,5 +298,7 @@ public void testTablesSystemTable()
         assertUpdate("DROP TABLE system_tables_test2");
         assertUpdate("DROP TABLE system_tables_test3");
         assertUpdate("DROP TABLE system_tables_test4");
+
+        computeActual("SELECT * FROM system.tables WHERE table_schema IN ('foo', 'bar')");
```

Comment:
Change this so that it actually returns something or at least test that the query returned no results.